README.md

```diff
@@ -68,8 +68,22 @@ zfs | Exposes [ZFS](http://open-zfs.org/) performance statistics. | [Linux](http
 
 ### Disabled by default
 
-The perf collector may not work by default on all Linux systems due to kernel
-configuration and security settings. To allow access, set the following sysctl
+`node_exporter` also implements a number of collectors that are disabled by default.  Reasons for this vary by
+collector, and may include:
+* High cardinality
+* Prolonged runtime that exceeds Prometheus` `scrape_interval` or `scrape_timeout`
+* Significant resource demands on the host
+
+You can enable additional collectors as desired by adding them to your init system's service description for `node_exporter`
+but caution is advised.  Enable at most one at a time, testing first on a non-production system, then by hand on a single
+production node.  You can check for functionality, cardinality, and execution time with
+
+```bash
+/usr/bin/time curl localhost:9100/metrics
+```
```

Comment:
I'm thinking it might be better to recommend doing this more "the Prometheus way".

Maybe something like this:

```
When enabling additional collectors, you should carefully monitor the change by observing the ` scrape_duration_seconds` metric to make sure it will not time out. In addition, monitor the `scrape_samples_post_metric_relabeling` to see the changes in cardinality.
```

Reply:
I'd gone with `curl` to avoid any circular dependency or catch-22, but you're right from a dogfood perspective.  I pushed an updated with an amended version of your suggestions, PTAL.